dp/generate_climos.py

```diff
@@ -124,62 +128,62 @@ def create_climo_files(outdir, input_file, t_start, t_end,
     date_range = '{},{}'.format(d2s(t_start), d2s(t_end))
     temporal_subset = cdo.seldate(date_range, input=input_file.filepath())
 
-    # Form climatological means over dependent variables
-    def climo_outputs(time_resolution):
+    # Form climatological means/standard deviations over dependent variables
+    def climo_outputs(time_resolution, operation):
         """Return a list of cdo operators that generate the desired climo outputs.
         Result depends on the time resolution of input file data - different operators are applied depending.
         If operators depend also on variable, then modify this function to depend on variable as well.
         """
         ops_by_resolution = {
-            'daily': ['ymonmean', 'yseasmean', 'timmean'],
-            'monthly': ['ymonmean', 'yseasmean', 'timmean'],
-            'yearly': ['timmean']
+            'daily': ['ymon' + operation, 'yseas' + operation, 'tim' + operation],
```

Comment:
You should check that `operation `has a valid value, and raise an exception with an informative message if not. Otherwise you will get a much harder to understand error a little later on.

Reply:
Not seeing the validation of `operation` here yet. ??

Reply:
Sorry, I do a check a little higher up in the outer method, I forgot to add it inside the nested method.